__tests__/app.test.js

```diff
@@ -41,4 +41,63 @@ describe("GET /api/topics", () => {
     .get("/api/topis")
     .expect(404)
   })
+})
+
+describe("GET /api/articles/article_id", () => {
+  test('200: returns an array of first article', () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({body}) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(1);
+        articles.forEach((obj) => {
+            expect(obj).toMatchObject({
+                author: expect.any(String),
```

Comment:
As we know which article we'll get back, I would rather assert against actual values here